src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -20,10 +20,26 @@
             <DataSource Persistence="ImplicitProjectFile" Label="Globals" HasConfigurationCondition="False" />
         </StringProperty.DataSource>
     </StringProperty>
-    <DynamicEnumProperty Name="TargetFrameworkMoniker" DisplayName="Target Framework Moniker" EnumProvider="TargetFrameworkMoniker" />
-    <StringProperty Name="TargetFrameworkIdentifier" Visible="False" />
-    <StringProperty Name="TargetFrameworkVersion" Visible="False" />
-    <StringProperty Name="TargetFrameworkProfile" Visible="False" />
+    <StringProperty Name="TargetFrameworkIdentifier" DisplayName="Target Framework Identifier">
```

Comment:
The HasConfigurationCondition="False" specifices that this value is not specific to configuration. So when writing out the value in a project file, it won't put it under a specific configuration.